cpp/ql/lib/qlpack.yml

```
@@ -6,4 +6,4 @@ extractor: cpp
 library: true
 upgrades: upgrades
 dependencies:
-  codeql/shared-ssa: "*"
+  codeql/shared-ssa: 0.0.1-dev
```

Here and below. I think we want the version without the `-dev` suffix since when this pack is published (and the shared packs are published), the packs won't have a `-dev` suffix. If we publish this pack with `-dev` in the version constraint, it can never be depended on by third parties since transitive dependencies can't be resolved.

```suggestion
  codeql/shared-ssa: 0.0.1
```